src/content/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration.mdx

```diff
@@ -2725,64 +2761,280 @@ Use these settings to toggle instrumentation types during agent startup.
+
+  If `true`, disables Javascript injection detection in IAST scans.
+  </Collapser>
+
+  <Collapser id="security-exclude_from_iast_scan-iast_detection_category-command_injection" title="security.exclude_from_iast_scan.iast_detection_category.command_injection">
+    <table>
+      <tbody>
+        <tr><th>Type</th><td>Boolean</td></tr>
+        <tr><th>Default</th><td>`false`</td></tr>
+        <tr><th>Environ variable</th><td>`NEW_RELIC_SECURITY_EXCLUDE_FROM_IAST_SCAN_IAST_DETECTION_CATEGORY_COMMAND_INJECTION`</td></tr>
+      </tbody>
+    </table>
+
+  If `true`, disables system command injection detection in IAST scans.
+  </Collapser>
+
+  <Collapser id="security-exclude_from_iast_scan-iast_detection_category-xpath_injection" title="security.exclude_from_iast_scan.iast_detection_category.xpath_injection">
+    <table>
+      <tbody>
+        <tr><th>Type</th><td>Boolean</td></tr>
+        <tr><th>Default</th><td>`false`</td></tr>
+        <tr><th>Environ variable</th><td>`NEW_RELIC_SECURITY_EXCLUDE_FROM_IAST_SCAN_IAST_DETECTION_CATEGORY_XPATH_INJECTION`</td></tr>
+      </tbody>
+    </table>
+
+  If `true`, disables XPATH injection detection in IAST scans.
+  </Collapser>
+
+  <Collapser id="security-exclude_from_iast_scan-iast_detection_category-ssrf" title="security.exclude_from_iast_scan.iast_detection_category.ssrf">
+    <table>
+      <tbody>
+        <tr><th>Type</th><td>Boolean</td></tr>
+        <tr><th>Default</th><td>`false`</td></tr>
+        <tr><th>Environ variable</th><td>`NEW_RELIC_SECURITY_EXCLUDE_FROM_IAST_SCAN_IAST_DETECTION_CATEGORY_SSRF`</td></tr>
+      </tbody>
+    </table>
+
+  If `true`, disables Sever-Side Request Forgery (SSRF) detection in IAST scans.
+  </Collapser>
+
+  <Collapser id="security-exclude_from_iast_scan-iast_detection_category-rxss" title="security.exclude_from_iast_scan.iast_detection_category.rxss">
+    <table>
+      <tbody>
+        <tr><th>Type</th><td>Boolean</td></tr>
+        <tr><th>Default</th><td>`false`</td></tr>
+        <tr><th>Environ variable</th><td>`NEW_RELIC_SECURITY_EXCLUDE_FROM_IAST_SCAN_IAST_DETECTION_CATEGORY_RXSS`</td></tr>
+      </tbody>
+    </table>
+
+  If `true`, disables Reflected Cross-Site Scripting (RXSS) detection in IAST scans.
+  </Collapser>
+
+  <Collapser id="security-scan_schedule-delay" title="security.scan_schedule.delay">
     <table>
       <tbody>
         <tr><th>Type</th><td>Integer</td></tr>
-        <tr><th>Default</th><td>`nil`</td></tr>
-        <tr><th>Environ variable</th><td>`NEW_RELIC_SECURITY_APPLICATION_INFO_PORT`</td></tr>
+        <tr><th>Default</th><td>`0`</td></tr>
+        <tr><th>Environ variable</th><td>`NEW_RELIC_SECURITY_SCAN_SCHEDULE_DELAY`</td></tr>
       </tbody>
     </table>
 
-  The port the application is listening on. This setting is mandatory for Passenger servers. Other servers should be detected by default.
+  Specifies the delay time (in minutes) before the IAST scan begins after the application starts.
```

Comment:
Indicates the time (in minutes) to wait before initiating the IAST scan after the application has started.